docs/drafts/results.html

```diff
@@ -73,6 +73,10 @@ <h2>Introduction</h2>
 				<a href="#sec-implementer-list"></a> lists all the implementations with the variant in parenthesis, if applicable.
 			</p>
 
+			<p>
+				The report tables also include a separate column (with the heading "Req?") with flag whether the test is required (i.e., testing a MUST statement) or optional (i.e., testing a SHOULD or MAY statement) for a full implementation conformance.
```

Comment:
As below, I think this shouldn't be a boolean — we should call it something like Priority or Importance or just MUST (as a header). Or even "Req" with no question mark.

Reply:
I kept "Req" (without the question mark). The horizontal real estate on the table is limited and a longer text would have required too much (horizontal) space.